### PR TITLE
Add non-zero exit to test/deploy.sh if Cx test failed

### PR DESCRIPTION
**Pull request by Daniel-VDM**
## Issue

Jenkins will not pick up a failure if script does not exit with a non-zero exit code.

### CODE REVIEW

File: test/deploy.sh [hunk 1]
```diff
@@ -208,6 +208,10 @@ if [ "$DOTEST" == "true" ]; then
    sleep 20
    check_result
    [ -e $RESULT_FILE ] && cat $RESULT_FILE
+   if [ "$err" = true ] ; then
```
Comment (LeoHChen) [inline test/deploy.sh]: can you use 
```"$err" == "true"```
 to be consistent with previous check.

Comment (Daniel-VDM) [reply test/deploy.sh]: Addresed in latest

**Comment by Daniel-VDM** [comment]
In my testing, waiting 20 seconds is *sometimes* not long enough for the Cx to go through. And infact this causes *some* Jenkins built tests to fail. Therefore, I always wait 25 seconds to check Cx on localnet.

**Comment by Daniel-VDM** [comment]
In my testing, waiting 20 seconds is *sometimes* not long enough for the Cx to go through.

**Comment by Daniel-VDM** [comment]
Jenkins, test this please.

**Pull request closed without merging**